convert_color_scheme.py

```diff
@@ -0,0 +1,135 @@
+"""Functionality for converting a color scheme to a "TerminalView" scheme."""
+import os
+import plistlib
+import sublime
+
+
+def hex_to_rgb(hexstring):
+    """Convert a string representing a hex color to an RGB tuple."""
+    return (int(hexstring[1:3], 16) / 255,
+            int(hexstring[3:5], 16) / 255,
+            int(hexstring[5:7], 16) / 255)
+
+
+def rgb_to_hex(rgb):
+    """Convert an RGB tuple to a hex string."""
+    return "#{:02x}{:02x}{:02x}".format(int(rgb[0] * 255),
+                                        int(rgb[1] * 255),
+                                        int(rgb[2] * 255))
+
+
+def norm2(vec3):
+    """Compute the squared norm of a three-dimensional vector."""
+    return vec3[0]**2 + vec3[1]**2 + vec3[2]**2
+
+
+def distance2(a, b):
+    """Compute the squared distance between two 3D vectors."""
+    return norm2((a[0] - b[0], a[1] - b[1], a[2] - b[2]))
```

Comment:
I did some tests with the `colormath`, and even the cie2000 version will choose the "wrong" color:
```
>>> delta_e_cie2000(convert_color(sRGBColor(0,1,1), LabColor), convert_color(sRGBColor(202/255,230/255,120/255), LabColor))
33.29340680537432
>>> delta_e_cie2000(convert_color(sRGBColor(0,1,1), LabColor), convert_color(sRGBColor(255/255, 204/255, 102/255), LabColor))
42.74613655513628
```
(first one is `#cae682`, second one is `#ffcc66`)
This makes me think that the reference colors are wrong. If you look at pictures of the color space you can see that the yellow region is very small compared to the green region. The reference color for yellow is sitting at the boundary of the space, maybe it should be placed in the middle of what looks yellowish.